models/articles-model.js

```diff
@@ -25,4 +25,14 @@ exports.selectArticlesComments = (id) => {
     return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`, [id]).then((data) => {
             return data.rows
         })
-    }
\ No newline at end of file
+}
+
+exports.addCommentByArticleId = (body, author, article_id) => {
+    const created_at = new Date()
```

Comment:
You shouldn't have to manually make a date and insert created_at, because pgsql does this for you automatically when you're inserting data and there's a date, it will assume the date is the current time. That's because your seed has this line - `created_at TIMESTAMP DEFAULT NOW()`

This is the same for votes, they default to zero because of the line:
`votes INT DEFAULT 0 NOT NULL `

Reply:
Oh I see, I'll remove that too!